aunt-leahs-app/src/components/select-box/select-box.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { InputLabel, MenuItem, FormControl, Select } from '@material-ui/core';
+
+class SelectBox extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            items: this.props.items || [],
+            selectedItem: this.props.items && this.props.items[0],
+        }
+    }
+
+    render() {
+        return <div>
```

Comment:
put the div on the next line, like this:

return (
     <div>
     ...
     </div>
);